### DISCUSSION TITLE
New Feature Request: Adjustable z index

### DISCUSSION THREAD

**Discussion by jjrohrer** [wire-elements/modal#260] [Ideas]
**The problem:** 
I had other stuff on my page set z-40, made the modal not appear on top. 

**Analysis:**
I found in 
`vendor/wire-elements/modal/resources/views/modal.blade.php` 
that the z index is set via tailwind to 10
 `class="fixed inset-0 z-10 overflow-y-auto"` 

**The Solution?:**
Since the modal creates a curtain, I think this needs to be in a setting in the config files as adjustable on a per project basis (and/or probably defaulting to z-50).

**Comment by thermiteplasma** [comment]
You can already do this by publishing the modal view and making the required edits to the blade file. 

`php artisan vendor:publish --tag=livewire-ui-modal-views`

**Comment by ju5t** [comment]
@thermiteplasma the thing with publishing the views is that you're not getting any updates either. Might not necessarily be a problem though, just putting it out there :)
